carbonmark-data/components/WebFonts/index.tsx

```diff
@@ -0,0 +1,12 @@
+export const WebFonts = () => {
```

Comment:
Also not sure if this will work in App Router.

Here is the new tooling they offer:
https://nextjs.org/docs/app/building-your-application/optimizing/fonts#google-fonts

Reply:
Removed the Webfont component in commit [5af5487](https://github.com/KlimaDAO/klimadao/pull/1471/commits/5af54870d3a1f8071c9ca337767762645d501709)
Created ticket #1485 to use the tooling. currently the fonts are directly imported through the global.css file